packages/app/src/elements/history-entry-dialog.ts

```diff
@@ -0,0 +1,121 @@
+import { translate as $l } from "@padloc/locale/src/translate";
+import { ItemHistory, Field } from "@padloc/core/src/item";
+import { confirm } from "../lib/dialog";
+import { Dialog } from "./dialog";
+import { html, css } from "lit";
+import { customElement } from "lit/decorators.js";
+import { repeat } from "lit/directives/repeat.js";
+
+@customElement("pl-history-entry-dialog")
+export class HistoryEntryDialog extends Dialog<ItemHistory, boolean> {
+    private _historyEntry: ItemHistory;
+
+    static styles = [
+        ...Dialog.styles,
+        css`
+            :host {
+                --pl-dialog-max-width: 40em;
+            }
+        `,
+    ];
+
+    // TODO: show vault changes
+
+    renderContent() {
+        if (!this._historyEntry) {
+            return html``;
+        }
+
+        const { fieldsChanged, date } = this._historyEntry;
+
+        return html`
+            <div class="padded vertical spacing layout">
+                <h1 class="big margined text-centering">${$l("History Entry")}</h1>
+                <h2 class="margined text-centering">${$l("Created at {0}", date as unknown as string)}</h2>
+
+                <div class="stretch"></div>
+
+                ${fieldsChanged.includes("name") ? this._showChangedNameContent() : null}
+                ${fieldsChanged.includes("tags") ? this._showChangedTagsContent() : null}
+                ${fieldsChanged.includes("fields") ? this._showChangedFieldsContent() : null}
+
+                <div class="horizontal evenly stretching spacing layout top-margined">
+                    <pl-button class="primary" @click=${() => this._restore()}>
+                        ${$l("Restore item to this version")}
```

Comment:
Let's keep this button label a bit shorter. Currently this looks a it weird on a narrow screen.